cmd/init.go

```diff
@@ -52,9 +52,7 @@ var providerProperties = map[string]ProviderProperties{
 // initCmd represents the init command
 var initCmd = &cobra.Command{
 	Use:   "init",
-	Short: "Generates a sample configuration file.",
-	Long: `Generates a sample configuration file and outputs it to the first argument.
-If there is no argument, configuration.yaml is used.`,
+	Short: "Generates configuration file.",
```

Comment:
Maybe mention it also generates the variable placeholders in `params.yaml`?
Something like:
"Refreshes YAML files and generates params.yaml configuration file."

Reply:
Makes sense, updated the text.